src/BinaryParsers/PEBinary/ProgramDatabase/Pdb.cs

```diff
@@ -268,24 +268,27 @@
~            {
~                enumSegments = this.CreateDiaTable<IDiaEnumSegments>();
~            }
~            catch (NotImplementedException) { }
 
             try

```

Comment:
>enumSegments [](http://example.com/codeflow?start=20&length=12)

we could move this outside the try, and check if that is null. if yes, return results.

This would reduce the branches.

Also, we could remove the comment in line 277, since we don't target netcoreapp2.0, we won't face that issue